Translations/make_translation.py

```diff
@@ -100,6 +100,7 @@ def getConstants():
     consants.append(("SymbolWatts", "W"))
     consants.append(("SymbolVolts", "V"))
     consants.append(("SymbolDC", "DC"))
+    consants.append(("SymbolNA", "N/A"))

```

Comment:
Do we expect this to be a constant across all languages worldwide? Might make more sense to move to language file..?